addons/mail_group/security/mail_group_security.xml

```diff
@@ -0,0 +1,126 @@
+                        ('mail_group_id.moderator_ids', 'in', user.id),
+                    '|',
+                    '|',
+                        ('mail_group_id.moderator_ids', 'in', user.id),
+                        ('mail_group_id.privacy', '=', 'public'),
+                        '&amp;',
+                            ('mail_group_id.privacy', '=', 'groups'),
+                            ('mail_group_id.group_public_id', 'in', [g.id for g in user.groups_id]),
+            ]
+        </field>
+        <field name="groups" eval="[(4, ref('base.group_user'))]"/>
+        <field name="perm_create" eval="True"/>
+        <field name="perm_read" eval="True"/>
+        <field name="perm_write" eval="True"/>
+        <field name="perm_unlink" eval="True"/>
+    </record>
+    <record id="mail_group_message_rule_administrator" model="ir.rule">
+        <field name="name">Mail Group Message: Administrator have access to all messages</field>
+        <field name="model_id" ref="model_mail_group_message"/>
+        <field name="domain_force">[]</field>
+        <field name="groups" eval="[(4, ref('mail_group.group_mailing_list_administrator'))]"/>
+        <field name="perm_create" eval="True"/>
+        <field name="perm_read" eval="True"/>
+        <field name="perm_write" eval="True"/>
+        <field name="perm_unlink" eval="True"/>
+    </record>
+    <record id="mail_group_member_rule_user" model="ir.rule">
+        <field name="name">Mail Group Member: Members are accessible only by moderators</field>
+        <field name="model_id" ref="model_mail_group_member"/>
+        <field name="domain_force">[('mail_group_id.moderator_ids', 'in', user.id), ('mail_group_id.moderation', '=', True)]</field>
+        <field name="groups" eval="[(4, ref('base.group_user'))]"/>
+        <field name="perm_create" eval="True"/>
+        <field name="perm_read" eval="True"/>
+        <field name="perm_write" eval="True"/>
+        <field name="perm_unlink" eval="True"/>
+    </record>
+    <record id="mail_group_member_rule_administrator" model="ir.rule">
+        <field name="name">Mail Group Member: Administrator have access to all members</field>
+        <field name="model_id" ref="model_mail_group_member"/>
+        <field name="domain_force">[]</field>
+        <field name="groups" eval="[(4, ref('mail_group.group_mailing_list_administrator'))]"/>
+        <field name="perm_create" eval="True"/>
+        <field name="perm_read" eval="True"/>
+        <field name="perm_write" eval="True"/>
+        <field name="perm_unlink" eval="True"/>
+    </record>
+    <record id="mail_group_moderation_rule_user" model="ir.rule">
+        <field name="name">Mail Group Moderation: Moderation rules are accessible only by moderators</field>
+        <field name="model_id" ref="model_mail_group_moderation"/>
+        <field name="domain_force">[('mail_group_id.moderator_ids', 'in', user.id), ('mail_group_id.moderation', '=', True)]</field>
```

Comment:
This means that if you remove the moderation flag, you cannot manage white list / blacklist ^^ I would use moderators or group responsible maybe ? To be sure we always have people that can modify moderation rules (same as members actually).

Reply:
(see comment above about the "Group Manager")